Live-Snap/Live-Snap/Controllers/ExportWallpaperViewController.swift

```diff
@@ -158,23 +160,28 @@ class ExportWallpaperViewController: UIViewController, PHLivePhotoViewDelegate {
         System.shared.appDelegate().pageViewController?.setViewControllers([cropWallpaperViewController], direction: .reverse, animated: true, completion: nil)
     }
     
+    func showSaveSuccessPopUpView() {
+        view.isUserInteractionEnabled = false
+        view.addSubview(saveSuccessBackgroundView)
+        view.addSubview(saveSuccessPopUpView)
+        
+        UIView.animate(withDuration: 0.25, delay: 0.0, options: .curveEaseOut, animations: {
+            self.saveSuccessPopUpView.center.y = self.view.center.y
+            self.saveSuccessBackgroundView.alpha = 0.75
+        }) { (finished: Bool) in
+            if finished {
+                self.view.isUserInteractionEnabled = true
+            }
+        }
+    }
```

Comment:
Should we add the ability for the user to exit the popup view? Only issue is if the user exists the popup and then taps save, it will save another photo to show the popup again

Reply:
since I think either way is fine, just leave as is so user has to tap a button on the popup